lib/keila_web/router.ex

```diff
@@ -151,11 +147,40 @@ defmodule KeilaWeb.Router do
     get "/c/:encoded_url/:recipient_id/:link_id/:hmac", TrackingController, :track_click
   end
 
-  scope "/api", KeilaWeb do
+  pipeline :api do
+    plug :accepts, ["json"]
+  end
+
+  scope "/api/v1", KeilaWeb do
```

Comment:
Just an idea, what do you think of putting this version `v1` based on a configuration of the project, so when we increase the number of the version api, the url updates as well.

I'm not really sure if this is possible